src/app/components/Inside/Player/Rating/Rating.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { getRating, setRating } from '../../../../services/MusicApi';
+import styles from '../Player.scss';
+
+class Rating extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rating: 0,
+    };
+    this.toggleRating = this.toggleRating.bind(this);
+    this.getRating = this.getRating.bind(this);
+  }
+
+  componentDidMount() {
+    this.getRating();
+  }
+
+  async getRating() {
+    const rating = await getRating('song', this.props.nowPlayingItem.id);
+    this.setState({
+      rating,
+    });
+  }
+
+  toggleRating() {
+    if (this.state.rating === 1) {
+      setRating('song', this.props.nowPlayingItem.id, -1).then(result =>
+        this.setState({
+          rating: result,
+        })
+      );
+    } else {
+      setRating('song', this.props.nowPlayingItem.id, this.state.rating + 1).then(result =>
+        this.setState({
+          rating: result,
+        })
+      );
+    }
+  }
+
+  render() {
+    let heartClass = 'far fa-heart';
```

Comment:
`fas`

Reply:
originally chose the far one because hollow was a bit more obviously not selected than just a grey one, but sure